src/test.rs

```diff
@@ -54,6 +54,12 @@ fn slices() {
         DeepSizeOf::deep_size_of(&array),
         size_of::<usize>() * 2 + size_of::<[u32; 64]>()
     );
+
+    let array: Box<[u32]> = vec![0; 1000].into_boxed_slice();
```

Comment:
Is this intended to check the DeepSizeOf on arrays? If that's the case, I think you want 

```suggestion
    let array: Box<[u32; 1000]> = vec![0; 1000].into_boxed_slice();
```

Right now it goes via the slice impl

Reply:
I believe the full correct code would be this, as `TryInto` is required for converting it into an array:
```rust
let array: Box<[u32; 1000]> = vec![0; 1000]
    .into_boxed_slice()
    .try_into()
    .unwrap();

assert_eq!(
    DeepSizeOf::deep_size_of(&array),
    size_of::<usize>() + size_of::<[u32; 1000]>()
);
```